topics/asciidoc.adoc

```diff
@@ -1,15 +1,21 @@
-[[asciidoc]]
+[asciidoc]
 == AsciiDoc
-http://asciidoc.org/[AsciiDoc^] is a text document format that you can use to write documentation, articles, books, web pages, and blogs. AsciiDoc files can be translated to many formats including HTML, PDF, EPUB, and man page.
+http://asciidoc.org/[AsciiDoc^] is a text document format that you can use to write documentation, articles, books, web pages, and blogs.
+AsciiDoc files can be translated to many formats including HTML, PDF, EPUB, and man page.
 
 The Red Hat documentation team uses http://asciidoctor.org/[Asciidoctor^] as part of the publishing toolchain.
 
-[[ccg-asciidoc-attributes]]
+[ccg-asciidoc-attributes]
 === AsciiDoc Attributes
 
-AsciiDoc attributes are variables you can use in common files to avoid hard-coding brand-specific information. Using attributes allows you to more easily share content between multiple brands. To use attributes, first define them in the relevant _brand_ AsciiDoc attribute file in the `common` directory. Then, include the _brand_ AsciiDoc attribute file in the `master.adoc` file for the relevant document. Finally, replace brand-specific terms with the variable names that you have defined, and use `ifdef` statements to indicate mark blocks of brand-specific content.
+AsciiDoc attributes are variables you can use in common files to avoid hard-coding brand-specific information.
+Using attributes allows you to more easily share content between multiple brands.
```

Comment:
Should we make this a numbered list/procedure?

Reply:
I agree, let's do it.